### PR TITLE
Alire.Utils.User_Input: add function to query user's name, email and GitHub login

### PR DESCRIPTION
**Pull request by Fabien-Chouteau**
Can be used to implement #455.

### CODE REVIEW

File: src/alire/alire-utils-user_input.adb [hunk 1]
```diff
@@ -201,4 +264,69 @@ package body Alire.Utils.User_Input is
...
+                               Validation => null));
+
+   -----------------------
+   -- User_GitHub_Login --
+   -----------------------
+
+   function User_GitHub_Login return String
+   is (Config_Or_Query_String (Config_Key => "user.github_login",
+                               Question   => "Your GitHub login",
+                               Default    => "githib-username",
```
Comment (mosteo) [inline src/alire/alire-utils-user_input.adb]: typo

**Review by mosteo — CHANGES_REQUESTED** [review]

File: src/alire/alire-utils-user_input.adb (hunk 1 above)
Comment (mosteo) [reply src/alire/alire-utils-user_input.adb]: (in "githib-username")

**Comment by Fabien-Chouteau** [comment]
Elaboration issue :(

Sorry I don't have time to investigate before the week-end.

**Comment by mosteo** [comment]
Hi Fabien, I took a stab at fixing the circular elab, I hope you don't mind.

**Comment by Fabien-Chouteau** [comment]
Thank you :+1:

**Review by mosteo — APPROVED** [review]

**Pull request merged**